README.md

```diff
@@ -103,6 +103,15 @@ In either case, you will want to have [Serverless] installed, eg. `npm install -
     pure [1, 2, 3]
   ```
 
+* Add `aeson` and `serverless-haskell` to package.yaml
+
+  ```yaml
+  dependencies:
+  - base >= 4.7 && < 5
+  - aeson
+  - serverless-haskell
+  ```
+
 * Use `sls deploy` to deploy the executable to AWS Lambda.
 
   The `serverless-haskell` plugin will build the package using Stack and upload
```

Comment:
Suggest adding a note about the _first_ build taking a long time here as well.

Reply:
Ok, take a look at the latest. I moved the local section along with the long build note above the deploy section, and added `export SLS_DEBUG=*` to the sample instructions for sls deploy, so people who are blindly following the guide will see the output.